### ISSUE TITLE
You need to rebase your fork!

### ISSUE THREAD

**Issue by orwell96** [Andrey2470T/Advanced-Trains-Optinal-Additional-#1]
Your fork is not up to date. You should rebase it onto the latest advtrains version.
If you don't know how to do this, tell me, i can explain and/or do it for you.

**Comment by Andrey2470T** [comment]
Hi. Sorry, i didn`t know here exists a chatting.
No, i don`t know. I`m trying to update now all the files and folders for 1.11 by hand, but i think this work is very pretty complicated as i have to watch all the files that were updated before. What to do with it and how to do it easier?

**Comment by Andrey2470T** [comment]
By the way, i have a question. How to delete/rename folders? And also, how to rename/delete the repository itself?

**Comment by orwell96** [comment]
renaming and deleting the repository works on github in the settings tab. but if you rename the repository, you need to clone it again (unless you want to mess around with "git remote")
renaming and deleting folders: just rename/delete them on your computer, run git add -A and git commit and git push
i'll open a pull request in the next days, you'll need to merge it.

**Comment by orwell96** [comment]
ok. I (or better you) have a problem...
The problem is that there's absolutely no association between the two repositories from the git perspective...
I tried to rebase, but this results in a complete mess...
you need to do the following:
1. go to https://github.com/orwell96/advtrains
2. click 'fork'
3. on your computer, open a terminal
4. navigate to the directory where you keep your programming projects
5. execute 'git clone https://github.com/Andrey2470T/advtrains'
-- At this point you have a copy of advtrains(vanilla) on your computer.
6. integrate all your changes into this version of advtrains
7. execute 'git add -A', 'git commit' (name your commit there) and 'git push' (enter github username and password)
-- done.
Then, i can help rebasing when new changes are available.

whenever you want to make changes to your fork:
1. execute 'git pull'
2. do your changes
3. execute 'git add -A', 'git commit' (name your commit there) and 'git push' (enter github username and password)

**Comment by Andrey2470T** [comment]
I would like to give you one important question as i am not good at Github: 
What do i need to clone my fork for?

**Comment by orwell96** [comment]
Git needs to keep a reference to the online repository to which it should upload changes to. (via git push). The easiest way to do this is via cloning it.

**Comment by Andrey2470T** [comment]
I`m trying to clone the repository, writing the command "git clone advtrains Advtrains-with-some-additions", but i`ve got like "fatal: repository "advtrains" doesn`t exist" (Translation to English from Russian). Probably do i need to do anything before that?

**Comment by orwell96** [comment]
The command is in my long post
'git clone https://github.com/Andrey2470T/advtrains'

**Comment by Andrey2470T** [comment]
Yeah, i can not execute "git add -A" how you wrote above. Here`s an error:
"fatal: not found the repository (or one of its directories up to the mount point /)
Stop searching for the end of the file system (as GIT_DISCOVERY_ACROSS_FILESYSTEM is not set)".

In advance, thank you.

**Comment by orwell96** [comment]
You need to execute all git commands insidd the directory of the repository.
The so-called "working directory" is written left of the prompt, and all commands you execute operate inside this directory
Are you using windows or linux?

**Comment by Andrey2470T** [comment]
I use Linux.What does "-A" mean after "git add" phrase?

**Comment by orwell96** [comment]
Add *everything* that was changed to the staging area
To get a terminal, navigate to the git repository folder in a file explorer, right-click anywhere and select 'Open terminal here', then execute the commands
